AGGREGATE.md

```diff
@@ -551,6 +551,56 @@ as when a trigger context ID is set.
 
 See [flexible_filtering.md](https://github.com/patcg-individual-drafts/private-aggregation-api/blob/main/flexible_filtering.md) for more details.
 
+### Optional: aggregatable buckets
+
+Aggregatable buckets is an optional feature that gives API callers the ability 
+to manage `L1` contribution budget distribution across different aggregatable
+buckets or types of conversions, addressing common challenges such as:
+
+- Allocating the privacy budget between different types of conversions
+  (e.g., biddable vs. non-biddable).
+- Distributing the budget across multiple SDKs to prevent any single SDK
```

Comment:
I think the use of the term `SDK` in this explainer document, without further elaboration, is confusing. Can we either give a more concrete example of this or find another example entirely?

Reply:
+1, the explainer should use web use cases.

Reply:
Good point, updated the use case with campaigns.